README.md

```diff
@@ -1,2 +1,44 @@
 # Free Code Camp - Applied InfoSec Challenges
 =============================================
+
+This is the repository for the first 11 freeCodeCamp [Information Security with HelmetJS Challenges](https://learn.freecodecamp.org/information-security-and-quality-assurance/information-security-with-helmetjs/).  
+For the challenges from the 12th on upwards, you have to clone [this one](https://github.com/freeCodeCamp/boilerplate-bcrypt). 
+
+If you haven't worked with **git** before, you might first want to read a few   
+[resources](http://try.github.io/) about it, or go the [Glitch](https://glitch.com/) 
+way as described on the above mentioned challenge introduction page.
+
+To clone this repository, go to a directory of your choice and run:
+```bash
+git clone https://github.com/freeCodeCamp/boilerplate-infosec.git
+cd boilerplate-infosec
+```
+But to be able to import it in Glitch or the coding platform of your choice,  
+you have to [create your own Repository on GitHub](https://help.github.com/articles/creating-a-new-repository/).  
+But before you are able to push to it, you have to remove the remote origin of
+freeCodeCamp,   
+rename the "gomix" branch to "master" and delete the gomix branch
+to have a clutter-free repository:
+```
```

Comment:
Same goes here - language hint and newlines.

Reply:
Please read about my comment on language hints below; to the newlines I agree.